### ISSUE TITLE
__dev__ and __prod__ don't work combined or in nested keys

### ISSUE THREAD

**Issue by lostfictions** [abhijithvijayan/wext-manifest-loader#12]
Hi there, thanks for this loader! I've noticed that when i do this

```
  "__firefox__applications": {
    "gecko": {
      "__dev__id": "dev-extension",
      "__prod__id": "<full extension id>"
    }
  },
```

`__dev__id` is not getting substituted correctly in development (and `__prod__id` doesn't get removed). An alternate formulation like this doesn't work either:

```
  "__dev____firefox__applications": {
    "gecko": {
      "id": "dev-extension"
    }
  },
  "__prod____firefox__applications": {
    "gecko": {
      "id": "<full extension id>"
    }
  },
```

Any ideas for how to resolve this problem? I'd like a stable temporary ID for opening devtools by default.

**Comment by abhijithvijayan** [comment]
Will look into this.

Never ran into this before.

**Comment by abhijithvijayan** [comment]
Bug confirmed.

**Comment by abhijithvijayan** [comment]
Fixed in `v2.3.0`

![Screenshot_20210221_132120](https://user-images.githubusercontent.com/34790378/108618901-b39f6580-7447-11eb-87dd-552b5d093068.png)

**Comment by lostfictions** [comment]
Amazing, thank you!
